Prettify the README (#46)
tenzir/platform: README.md

<a target="_blank" href="https://docs.tenzir.com">
<p align="center">
<img src="./assets/tenzir-white.svg#gh-dark-mode-only" width="60%" alt="Tenzir">
<img src="./assets/tenzir-black.svg#gh-light-mode-only" width="60%" alt="Tenzir">
</p>
</a>

<h3 align="center">
Data Pipelines for Security Teams
</h3>
</p>

## What is Tenzir?

> Tenzir is the data pipeline engine for security teams.
> 

Reviewer: ```suggestion
```

We have that already above. Probably not necessary to duplicate.

If you need to collect, parse, shape, normalize, aggregate, store, query, and
route security telemetry data at scale, you'll love how our pipelines manage
your dataflows. Tenzir makes it easy to quickly onboard and store data from
numerous sources, reduce data volumes to optimize cloud and data costs, and
execute detections and run analytics in-stream.

## Get started

This repository hosts the on-premise version of the Tenzir Platform and a
command-line utility for managing workspaces and nodes on the Tenzir Platform.

If you are a customer of the **Sovereign Edition**, deploy the Tenzir Platform
locally by following our [Deploy the platform][guide] setup guide step by step.

> If you have access to the full source code of the Tenzir Platform and want to
> develop it or make modifications, we recommend starting with the [Development
> version][dev-version] instead that ties into the infrastructure behind
> [app.tenzir.com][app].

Reviewer: ```suggestion
If you are a developer and want to work on the code base of the Tenzir Platform,
we recommend starting with the [Development
version][dev-version] instead that ties into the infrastructure behind
[app.tenzir.com][app].
```

> [!NOTE]
> To get more information about the **Sovereign Edition**, please [contact

> > sales](https://tenzir.com/pricing). Visit [app.tenzir.com][app] for a free,

Reviewer: ```suggestion
> sales](sales@tenzir.com). Visit [app.tenzir.com][app] to enjoy the free,
```

> > cloud-hosted version.

Reviewer: ```suggestion
> cloud-hosted version of the Tenzir Platform.
```
